fv3core/Makefile

```diff
@@ -38,8 +37,7 @@ PYTHON_FILES = $(shell git ls-files | grep -e 'py$$' | grep -v -e '__init__.py')
 PYTHON_INIT_FILES = $(shell git ls-files | grep '__init__.py')
 TEST_DATA_TARFILE=dat_files.tar.gz
 TEST_DATA_TARPATH=$(TEST_DATA_HOST)/$(TEST_DATA_TARFILE)
-CORE_TAR=$(SARUS_FV3CORE_IMAGE).tar
-MPIRUN_CALL ?=mpirun -np $(NUM_RANKS)
+MPIRUN_CALL ?=mpirun -np $(NUM_RANKS) --oversubscribe --mca btl_vader_single_copy_mechanism=none
```

Comment:
Does this matter if we're running on gce vm vs. circle CI vm?

Reply:
These are the commands for openmpi-based mpirun, it shouldn't matter. The only place we use mpich is Piz Daint, and there we pass this explicitly.

Reply:
Turns out this was an issue, looks like GCE uses mpich. I've added handling for this so the .jenkins-based workflow removes these flags if in the "virtualenv" setup.